test/goldens/end2end/url_launcher-6.0.3.json_report.md

```diff
@@ -13,7 +13,6 @@
 
 ### [*] 10/10 points: Package has an example
 
-* Found example at: `example/lib/main.dart`
```

Comment:
Should this line have stayed?

Reply:
No, as discussed. Since we are adding more issue types it doesn't make sense to have a bullet point with the positive case in between the other issues.